app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -58,6 +58,7 @@ def display_name
   def show_container(record, controller_name, display_name)
     return if record_no_longer_exists?(record)
 
+    @current_page ||= 1
```

Comment:
@zeari why do we need to initialize ``` @current_page ``` explicitly here, this should have been done by get_view method in application_controller https://github.com/ManageIQ/manageiq/blob/master/app/controllers/application_controller.rb#L1566